### ISSUE TITLE
Can't move between tabs using keyboard shortcuts after moving to a tab using Quick Tabs

### ISSUE THREAD

**Issue by infokiller** [babyman/quick-tabs-chrome-extension#101]
When I invoke the extension and select another tab (either via the keyboard or mouse), I can't use the Chrome keyboard shortcuts to move between tabs (Ctrl+PgUp/PgDn). Defocusing and refocusing the Chrome window fixes that, but it's quite annoying to do every time. I reproduced this using an incognito window with just Quick Tabs enabled.
Chrome version:
46.0.2490.80 (Official Build) (64-bit)

**Comment by pearj** [comment]
It seems that it doesn't set the focus page to the page after you switch tabs?  I've noticed all keyboard shortcuts (including arrow keys to scroll the page) don't work, except the shortcut to open the extension again.

So after switching tabs, if you press Ctrl+E (the shortcut I assigned) again and then Esc, you don't need to use the mouse. But it's a bit annoying.

Version 52.0.2743.82 m (64-bit) Windows
